internal/manifest/fixtures/maven/with-scope.xml

```diff
@@ -1,10 +1,14 @@
 <project>
+  <groupId>com.mycompany.app</groupId>
+  <artifactId>my-app</artifactId>
+  <version>1.0</version>
+
   <dependencies>
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
       <version>4.12</version>
-      <scope>test</scope>
```

Comment:
@michaelkedar I think we should fix the resolver not to ignore `test` dependencies. `junit:junit` is not in the resolve graph. :(